client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-setup-dialog.js

```diff
@@ -146,7 +146,7 @@ class PaymentMethodStripeSetupDialog extends Component {
 				isVisible
 			>
 				<div className="stripe__method-edit-header">
-					{ translate( 'Take credit card payments with Stripe' ) }
+					{ translate( 'Accept credit card payments with Stripe' ) }
```

Comment:
:disappointed: 1 existing translations will be lost with this change.

Reply:
Losing one translation is probably not a big deal in this case, I'd disregard.

Reply:
I think it's worth it. We use _Accept_ elsewhere and it's a more positive word than _Take_.